src/devices/Max7219/Max7219.cs

```diff
@@ -0,0 +1,323 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Device.Gpio;
+using System.Device.Spi;
+
+namespace Iot.Device
+{
+    public class Max7219 : IDisposable
+    {
+        private SpiDevice _spiDevice;
+
+        /// <summary>
+        /// Number of digits Register per Module
+        /// </summary>
+        const int NumDigits = 8;
+
+        /// <summary>
+        /// Internal buffer used to write to the devices.
+        /// </summary>
+        private readonly byte[] _writeBuffer;
+
+        /// <summary>
+        /// A Buffer that contains the values of the digits registers per device
+        /// </summary>
+        private readonly byte[,] _buffer;
```

Comment:
It is not obvious to me what is the difference between the write buffer and the _buffer, can we consider renaming this one and adding more helpful comment?